# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocdecode/hotfix-cc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocDecode-rocm-6.3.3/CMakeLists.txt.vanilla	2025-04-03 18:07:20.893796300 +0200
+++ rocDecode-rocm-6.3.3/CMakeLists.txt	2025-04-03 18:07:30.131297278 +0200
@@ -78,7 +78,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH} ${ROCM_PATH}/hip)
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode Build Type
 if(NOT CMAKE_BUILD_TYPE)
--- rocDecode-rocm-6.3.3/samples/videoToSequence/CMakeLists.txt.vanilla	2025-04-03 18:05:17.355987206 +0200
+++ rocDecode-rocm-6.3.3/samples/videoToSequence/CMakeLists.txt	2025-04-03 18:05:24.664363775 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
--- rocDecode-rocm-6.3.3/samples/videoDecodeRGB/CMakeLists.txt.vanilla	2025-04-03 18:05:17.359453369 +0200
+++ rocDecode-rocm-6.3.3/samples/videoDecodeRGB/CMakeLists.txt	2025-04-03 18:05:30.169550518 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
--- rocDecode-rocm-6.3.3/samples/videoDecodePerf/CMakeLists.txt.vanilla	2025-04-03 18:05:17.362378959 +0200
+++ rocDecode-rocm-6.3.3/samples/videoDecodePerf/CMakeLists.txt	2025-04-03 18:05:34.476472414 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
@@ -99,4 +99,4 @@
     if (NOT Threads_FOUND)
         message(FATAL_ERROR "-- ERROR!: Threads Not Found! - please insatll Threads!")
     endif()
-endif()
\ No newline at end of file
+endif()
--- rocDecode-rocm-6.3.3/samples/videoDecodeMultiFiles/CMakeLists.txt.vanilla	2025-04-03 18:05:17.365319499 +0200
+++ rocDecode-rocm-6.3.3/samples/videoDecodeMultiFiles/CMakeLists.txt	2025-04-03 18:05:50.782276220 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
@@ -92,4 +92,4 @@
     if (NOT ROCDECODE_FOUND)
         message(FATAL_ERROR "-- ERROR!: rocDecode Not Found! - please install rocDecode!")
     endif()
-endif()
\ No newline at end of file
+endif()
--- rocDecode-rocm-6.3.3/samples/videoDecodeMem/CMakeLists.txt.vanilla	2025-04-03 18:05:17.368253540 +0200
+++ rocDecode-rocm-6.3.3/samples/videoDecodeMem/CMakeLists.txt	2025-04-03 18:05:57.653326214 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
@@ -92,4 +92,4 @@
     if (NOT ROCDECODE_FOUND)
         message(FATAL_ERROR "-- ERROR!: rocDecode Not Found! - please install rocDecode!")
     endif()
-endif()
\ No newline at end of file
+endif()
--- rocDecode-rocm-6.3.3/samples/videoDecodeBatch/CMakeLists.txt.vanilla	2025-04-03 18:05:17.371208231 +0200
+++ rocDecode-rocm-6.3.3/samples/videoDecodeBatch/CMakeLists.txt	2025-04-03 18:06:01.475313441 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
@@ -103,4 +103,4 @@
     if (NOT Threads_FOUND)
         message(FATAL_ERROR "-- ERROR!: Threads Not Found! - please insatll Threads!")
     endif()
-endif()
\ No newline at end of file
+endif()
--- rocDecode-rocm-6.3.3/samples/videoDecode/CMakeLists.txt.vanilla	2025-04-03 18:07:20.890840019 +0200
+++ rocDecode-rocm-6.3.3/samples/videoDecode/CMakeLists.txt	2025-04-03 18:07:26.557186490 +0200
@@ -36,7 +36,7 @@
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../../cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
-set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
+#set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/clang++)
 
 # rocDecode sample build type
 set(DEFAULT_BUILD_TYPE "Release")
